### PR TITLE
updating apps directory size and test instance type

### PR DESCRIPTION
**Pull request by jagdeep-sonar**
https://onejira.imperva.com/browse/SD-1538 duplicates https://onejira.imperva.com/browse/EDSF-491

### CODE REVIEW

File: modules/aws/sonar-base-instance/setup.tftpl [hunk 1]
```diff
@@ -92,7 +92,7 @@ function create_lvm() {
     # Create logical volumes with lowercase names
     lvcreate --name logs -L 20G $volume_group_name
     lvcreate --name local -L 20G $volume_group_name
-    lvcreate --name apps -L 30G $volume_group_name
+    lvcreate --name apps -L 40G $volume_group_name
```
Comment (eytannnaim) [inline modules/aws/sonar-base-instance/setup.tftpl:95]: I would suggest increasing the default total size of the default storage size in the base sonar instance (And maybe from all the relevant examples as well.)

File: examples/aws/poc/dsf_deployment/variables.tf [hunk 2]
```diff
@@ -246,7 +246,7 @@ variable "agentless_gw_hadr" {
 
 variable "hub_instance_type" {
   type        = string
-  default     = "r6i.xlarge"
+  default     = "r6i.2xlarge"
```
Comment (eytannnaim) [inline examples/aws/poc/dsf_deployment/variables.tf:249]: You should also change these in the sonar base module. The reason we put relatively small instances is to make it cheaper for poc environments which aren't being used by customers.

File: modules/aws/sonar-base-instance/setup.tftpl (hunk 1 above)
Comment (jagdeep-sonar) [reply modules/aws/sonar-base-instance/setup.tftpl:95]: in examples, disk_size is already 250gb for hub and 150gb for agentless_gw. how much do you suggest to increase this?

variable "hub_ebs_details" {
  type = object({
    disk_size        = number
    provisioned_iops = number
    throughput       = number
  })
  description = "DSF Hub compute instance volume attributes. More info in sizing doc - https://docs.imperva.com/bundle/v4.10-sonar-installation-and-setup-guide/page/78729.htm"
  default = {
    disk_size        = 250
    provisioned_iops = 0
    throughput       = 125
  }
}
variable "agentless_gw_ebs_details" {
  type = object({
    disk_size        = number
    provisioned_iops = number
    throughput       = number
  })
  description = "DSF Agentless Gateway compute instance volume attributes. More info in sizing doc - https://docs.imperva.com/bundle/v4.10-sonar-installation-and-setup-guide/page/78729.htm"
  default = {
    disk_size        = 150
    provisioned_iops = 0
    throughput       = 125
  }
}

File: examples/aws/poc/dsf_deployment/variables.tf (hunk 2 above)
Comment (jagdeep-sonar) [reply examples/aws/poc/dsf_deployment/variables.tf:249]: on it.

File: modules/aws/sonar-base-instance/setup.tftpl (hunk 1 above)
Comment (eytannnaim) [reply modules/aws/sonar-base-instance/setup.tftpl:95]: I really can't say. I think you should take it with @jsonar-cpapke , decide about the values and when you finish we can give the feedback.

Of course, if you have a specific question I can answer, but we need you to try and own this.

And to your question. i think that if you increase the app share by 10G you should also increase the default size for the entire disk by 10G.

Comment (jsonar-cpapke) [reply modules/aws/sonar-base-instance/setup.tftpl:95]: Considering the sizes would be too small even for small test production systems, I don't really see any reason to try accommodate any systems that aren't designed to be quick tests with our default values.

**Review by jsonar-cpapke — APPROVED** [review]

**Review by eytannnaim — APPROVED** [review]

**Pull request merged**